pkg/util/file.go

```diff
@@ -48,3 +48,14 @@ func WriteFileToLog(logger *capnslog.PackageLogger, path string) {
 
 	logger.Infof("Config file %s:\n%s", path, string(contents))
 }
+
+func FileExists(path string) (bool, error) {
```

Comment:
tests?

Reply:
`FileExists()` method was removed after the comments - https://github.com/rook/rook/pull/3606#discussion_r314133260 . So tests won't be required for this method. 
I've another [ticket](https://github.com/rook/rook/issues/3610) where I'll be adding all the relevant tests related to StorageClassDeviceSets. let me know if that works.